docs/2_installation.rst

```diff
@@ -75,6 +75,70 @@ Many people have different configurations for their development and production e
 and running the application with misconfigured settings can prevent security features from working.
 
 
+Version 6 breaking changes and upgrading from django-axes version 5
+-------------------------------------------------------------------
+
+If you have not specialized ``django-axes`` configuration in any way
+you do not have to update any of the configuration.
+
+The instructions apply to users who have configured ``django-axes`` in their projects
+and have used flags that are deprecated. The deprecated flags will be removed in the future
+but are compatible for at least version 6.0 of ``django-axes``.
+
+The following flags and configuration have changed:
+
+``django-ipware`` has become an optional dependency.
+To keep old behaviour, use ``pip install django-axes[ipware]``
+in your install script or use ``django-axes[ipware]``
+in your requirements file(s) instead of plain ``django-axes``.
+The new ``django-axes`` package does not include ``django-ipware`` by default
+but does use ``django-ipware`` if it is installed
+and no callables for IP address resolution are configured
+with the ``settings.AXES_CLIENT_IP_CALLABLE`` configuration flag.
+
+``django-ipware`` related flags have changed names.
+The old flags have been deprecated and will be removed in the future.
+To keep old behaviour, rename them in your settings file:
+
+- ``settings.AXES_PROXY_ORDER`` is now ``settings.AXES_IPWARE_PROXY_ORDER``,
+- ``settings.AXES_PROXY_COUNT``  is now ``settings.AXES_IPWARE_PROXY_COUNT``,
+- ``settings.AXES_PROXY_TRUSTED_IPS`` is now ``settings.AXES_IPWARE_PROXY_TRUSTED_IPS``, and
+- ``settings.AXES_META_PRECEDENCE_ORDER`` is now ``settings.AXES_IPWARE_META_PRECEDENCE_ORDER``.
+
+``settings.AXES_LOCKOUT_PARAMETERS`` configuration flag has been added which supersedes the following configuration keys:
+
+- ``settings.AXES_ONLY_USER_FAILURES``,
+- ``settings.AXES_LOCK_OUT_BY_COMBINATION_USER_AND_IP``,
+- ``settings.AXES_LOCK_OUT_BY_USER_OR_IP``, and
+- ``settings.AXES_USE_USER_AGENT``.
+
+To keep old behaviour with the new flag, configure the following:
+
+- If you did not use any flags, use ``settings.AXES_LOCKOUT_PARAMETERS = ["ip_address"]``,
+- If you used ``settings.AXES_ONLY_USER_FAILURES``, use ``settings.AXES_LOCKOUT_PARAMETERS = ["username"]``,
+- If you used ``settings.AXES_LOCK_OUT_BY_COMBINATION_USER_AND_IP``, use ``settings.AXES_LOCKOUT_PARAMETERS = [["username", "ip_address"]]``,
+- If you used ``settings.AXES_LOCK_OUT_BY_USER_OR_IP``, use ``settings.AXES_LOCKOUT_PARAMETERS = ["username", "ip_address"]``, and
+- If you used ``settings.AXES_USE_USER_AGENT``, add ``"user_agent"`` to your list(s) of lockout parameters.
```

Comment:
As for use_user_agent, it should be clarified:
1. If other flags were not used, then you need to configure like this
```python
 settings.AXES_LOCKOUT_PARAMETERS = [["ip_address", "user_agent"]]
 ```
 2. If there was a flag ONLY_USER_FAILURES, then 
 
```python
 settings.AXES_LOCKOUT_PARAMETERS = [["username", "user_agent"]]
 ```
 3. If there was a flag AXES_LOCKOUT_BY_USER_OR_IP:
 
```python
 settings.AXES_LOCKOUT_PARAMETERS = [["ip_address", "user_agent"], "username"]
 ```
 4. If there was a flag AXES_LOCKOUT_BY_COMBINATION_USER_AND_IP, than
  
```python
 settings.AXES_LOCKOUT_PARAMETERS = [["ip_address", "user_agent", "username"]]
 ```
 This needs to be clarified, because how it worked before is not at all obvious

Reply:
Default for `AXES_USE_USER_AGENT` was previously `False` and the defaults as they are defined today are described in https://github.com/jazzband/django-axes/blob/d463787e343f80234e392435920a51a47bfe19a9/axes/conf.py#L17

`user_agent` is only used if `AXES_USE_USER_AGENT` is `True`, and it is not included by default in the settings?

Reply:
Updated, does this look better now?